aws/resource_aws_ssm_maintenance_window.go

```diff
@@ -14,6 +14,9 @@ func resourceAwsSsmMaintenanceWindow() *schema.Resource {
 		Read:   resourceAwsSsmMaintenanceWindowRead,
 		Update: resourceAwsSsmMaintenanceWindowUpdate,
 		Delete: resourceAwsSsmMaintenanceWindowDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
Can you please add the covering acceptance testing for importing? In `TestAccAWSSSMMaintenanceWindow_basic` under the `Steps: []resource.TestStep{`:

```go
{
	ResourceName:      "aws_ssm_maintenance_window.foo",
	ImportState:       true,
	ImportStateVerify: true,
},
```

Thanks!

Reply:
Added, thanks for the snippet!